### ISSUE TITLE
PAUP fails if paup binary is not called "paup"

### ISSUE THREAD

**Issue by SimonGreenhill** [FePhyFoFum/quartetsampling#4]
If the binary for PAUP* is not called `paup`, then the following crash happens:

```
Traceback (most recent call last):
  File "quartet_sampling.py", line 304, in <module>
    main()
  File "quartet_sampling.py", line 264, in main
    pool.map(process_replicate_paup, replicates)
  File "./lib/python3.6/multiprocessing/pool.py", line 260, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "./lib/python3.6/multiprocessing/pool.py", line 608, in get
    raise self._value
FileNotFoundError: [Errno 2] No such file or directory: 'paup'
```

This is because [L643 of rep_data](https://github.com/FePhyFoFum/quartetsampling/blob/master/pysrc/rep_data.py#L643) calls `paup` explicitly, rather than using the value in `args`.

**Comment by jbpease** [comment]
This has been corrected now in version 1.1.1, the --paup-executable flag should now work properly

**Comment by SimonGreenhill** [comment]
Looks good, thanks!
